strawberryfields/ops.py

```diff
@@ -463,15 +456,17 @@ def _apply(self, reg, backend, **kwargs):
             reg (Sequence[int]): subsystem indices the operation is
                 acting on (this is how the backend API wants them)
             backend (BaseBackend): backend to execute the operation
+
+        Returns:
+            array[Number] or None: Measurement results, if any. shape == (len(reg), shots).
         """
-        raise NotImplementedError(
-            'Missing direct implementation: {}'.format(self))
+        raise NotImplementedError('Missing direct implementation: {}'.format(self))
```

Comment:
lol obviously people's editors have different standards for acceptable line length. @josh146 can we just go ahead and pick one and use `black` to enforce it?

Reply:
On GitHub, the displayed linewidth varies depending on the browser, OS, and monitor display settings, and can be anywhere from 100-130 chars per line.

I suggest 100 line width, if everyone agrees.

Reply:
:+1:

Reply:
100 chars is good.